pkg/certificate/extensions.go

```diff
@@ -18,16 +18,36 @@ import (
 	"crypto/x509/pkix"
 	"encoding/asn1"
 	"errors"
+	"fmt"
 )
 
 var (
+	// BEGIN: Deprecated
```

Comment:
You may need to comment these individually as `// Deprecated: ...` if you want LSPs to pick these up as deprecated - https://github.com/golang/go/wiki/Deprecated